### ISSUE TITLE
App notify_push not enabled by default (on migration)

### ISSUE THREAD

**Issue by ManOki** [nextcloud/all-in-one#1702]
### Steps to reproduce
1. Migrate an existing nextcloud instance to aio
2. Check admin webpage or `php occ config:app:get notify_push enabled`, if notify_push is enabled. (I don't know if it was enabled before or not.)

### Expected behavior 
App notify_push should be enabled [by default](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L406).

### Actual behavior 
It is not enabled.

Actually, it is installed but has no config value for `enabled` (which means disabled), so [this check in line 409](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L409) fails. Should be `!= "yes"`.

Corresponding output of `php occ config:list`:

```
       "notify_push": {
           "base_endpoint": "https:\/\/yourdomain.com\/push"
       },                                                                    
```

I checked `entrypoint.sh` and there are more lines like this, see [400](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L400)/[402](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L402), [422](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L422), [482](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L482), [501](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L501), [531](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L531), [567](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L567), [574](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L574), and [581](https://github.com/nextcloud/all-in-one/blob/92b271c3e5f3d6a76889b94c85b89445881e30b0/Containers/nextcloud/entrypoint.sh#L581)

**Comment by ManOki** [comment]
Workaround is of course manually enable app via `php occ app:enable notify_push` ;)

**Comment by szaimen** [comment]
Hi, thanks for the report! The suggested fix looks good to me! Do you mind creating a PR with the fix? Otherwise I'll create a PR probably tomorrow. Thanks!

**Comment by szaimen** [comment]
PR in https://github.com/nextcloud/all-in-one/pull/1708

**Comment by szaimen** [comment]
This is now released with v4.2.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
